expanalysis/jspsych_utils.py

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
+"""
```

Comment:
You would want jspych utils to be a module within expfactory-analysis, eg:

expanalysis/
   experiments/
       jspsych.py
   utils.py

That way the user would do something like

      from expanalysis.experiments.jspsych import ...